lib/round-tracker.js

```diff
@@ -0,0 +1,78 @@
+import { createMeridianContract } from './meridian/contract.js'
+
+export async function createRoundGetter (pgClient) {
+  const contract = await createMeridianContract()
+
+  let sparkRound
+  const updateSparkRound = async (meridianRound) => {
+    sparkRound = await mapCurrentMeridianRoundToSparkRound({
+      meridianContractAddress: contract.address,
+      meridianRound: BigInt(meridianRound),
+      pgClient
+    })
+    console.log('SPARK round started: %s', sparkRound)
+  }
+
+  contract.on('RoundStart', updateSparkRound)
```

Comment:
`updateSparkRound` is an `async` function and I'm not aware of `ethers` having some logic to properly handle async event handlers. Therefore I think this needs a `.catch`

Reply:
Great catch! (No pun intended 🤣)